README.md
# SpaceHub-back-end

history.txt
    1  nano .zshrc
    2  brew install zsh-syntax-highlighting\n
    3  git clone git://github.com/zsh-users/zsh-autosuggestions $ZSH_CUSTOM/plugins/zsh-autosuggestions\n
    4  atom
    5  brew help
    6  $ cowsay hello
    7  brew install cowsay
    8  cowsay -f dragon hello
    9  cowsay dragon roar
   10  cowsay -f dragon roar
   11  cowsay -f help
   12  cowhide
   13  cowsay "hello"
   14  lkdtt dlsk 
   15  cowsay 'help'
   16  cowsay
   17  cowsay help
   18  brew install git
   19  which git
   20  git --version
   21  whoami
   22  git config --global user.name 'Jeff Miguel Hernandez'
   23  git config --global user.email 'jeffhernandez89@gmail.com\n'
   24  brew install zsh
   25  atom ~/Desktop/test.js
   26  zsh --version
   27  node ~/Desktop/test.js
   28  rm ~/Desktop/test.js
   29  exho $HOME
   30  echo $home
   31  mkdir galvanize && \\ncd galvanize && \\nmkdir galvanize && \\ncd galvanize && \\nmkdir unit-1 && mkdir unit-2 && mkdir unit-3 && mkdir unit-4 && mkdir projects && \\nls -lamkdir galvanize && \\ncd galvanize && \\nmkdir unit-1 && mkdir unit-2 && mkdir unit-3 && mkdir unit-4 && mkdir projects && \\nls -la
   32  mkdir galvanize && \\ncd galvanize && \\nmkdir unit-1 && mkdir unit-2 && mkdir unit-3 && mkdir unit-4 && mkdir projects && \\nls -la
   33  .exit
   34  mwd
   35  open
   36  echo $HOME
   37  sh -c "$(curl -fsSL https://raw.githubusercontent.com/robbyrussell/oh-my-zsh/master/tools/install.sh)"\n
   38  ZSH_THEME="agnoster"
   39  $HOME
   40  atom .oh-my-zsh
   41  /Users/jeffhernandez/Downloads/fonts-master/install.sh; exit;
   42  mv galvanize/* .
   43  rmdir galvanize
   44  cd desktop
   45  mkdir firstProject
   46  cd 
   47  cd [firstProject]
   48  cd /
   49  ls -|
   50  touch secondPage.html
   51  touch styles.css
   52  open secondPage.html
   53  touch stylesTwo.css
   54  open test.html
   55  touch divTest.html
   56  cd ls
   57  cat styles.css
   58  cd -ls
   59  rm myGif.gif
   60  rm myGif.mp4
   61  mv test.html firstPage.html
   62  rm Tn9uqwk.png
   63  mv styles.css stylesTwo.css
   64  cat adult.gif
   65  1;2c1;2c1;2c1;2c1;2c1;2c1;2c1;2c1;1;112;112;1;0x1;2c
   66  cd jeffhernandez/firstProject
   67  cd galv
   68  cd Pictures
   69  vim funny-jokes.md
   70  touch vimTest.md
   71  vim vimTest.md
   72  cd -
   73  cp cli-notes.md cli-notes-2.md
   74  mv cli-notes-2.md cli_commands.md
   75  rm notes
   76  cd ...
   77  cd jeffhernandez/galvanize
   78  touch cli_notes.md
   79  vim cli_notes.md
   80  mv cli_notes.md notes
   81  cp cli_notes.md cli_notes_2
   82  mv cli_notes_2 cli_commands.md
   83  rm -r notes
   84  cp stylesTwo.css stylesOne.css
   85  mv vimTest.md vimtestTwo
   86  mv vimTest.md vimtestTwo.md
   87  touch test.md
   88  mv test.md testTwo.md
   89  cd Documents
   90  atom indext.html
   91  git commit -m "Initial commit"
   92  cp firstpage.html index.html
   93  git commit -m "initial try"
   94  git add index.html
   95  mkdir git-example
   96  rm -r .git
   97  cd git-example
   98  git commit -m "third try"
   99  git remote add git@github.com:jhernandez89/myTest.git
  100  git remote add origin git@github.com:jhernandez89/myTest.git
  101  git remote add origin https://github.com/jhernandez89/myTest.git
  102  rm origin
  103  git remote add originTwo https://github.com/jhernandez89/myTest.git
  104  git checkout -b
  106  brew install homebrew/emacs/powerline-emacs
  107  git checkout tesst
  109  brew install powerline
  110  git checkout landing-page -b
  111  git commit -m "first try"
  112  git commit -m "second try"
  113  git push origin landing-page
  114  git push originTwo landing-page
  115  git remote rename originTwo origin
  116  git push -h
  117  git push -v
  118  git add -h
  119  brew install powerline fonts
  120  brew install https://github.com/Lokaltog/powerline/tarball/develop
  121  brew install python
  122  python --version
  123  pip install https://github.com/Lokaltog/powerline/tarball/develop
  124  pip install powerline
  125  ....
  126  jeffhernandez
  127  cd local/lib
  128  cd python2.7
  129  cd site-packages
  130  ~/.vimrc:
  131  cd ~/vimrc
  132  cd ~/.vimrc
  133  ~/.vimrc
  134  powerline
  135  open powerline
  136  ...........
  137  cd ~
  138  cd usr
  139  find powerline
  140  zshconfig
  141  . .zshrc
  142  source .zshrc
  143  rm .git
  144  git commit -m "first save"
  145  git commit -m "second save"
  146  git push origin gitPractice
  147  git remote add origin https://github.com/jhernandez89/testRepo.git

seeds/04_room.js

exports.seed = function(knex, Promise) {
  // Deletes ALL existing entries
  return knex('room').del()
    .then(function () {
      // Inserts seed entries
      return knex('room').insert([
        { name: "Orange", available: false, capacity: 40, hour_rate: 42.24, day_rate: 70.30, img_url: "http://ghk.h-cdn.co/assets/cm/15/11/54ff82285cf02-lving-room-green-gold-modern-de.jpg", venue_id: 2 },
        { name: "Black", available: false, capacity: 30, hour_rate: 18.53, day_rate: 75.34, img_url: "https://images-na.ssl-images-amazon.com/images/G/01/img16/home/content-grid/shop-by-room/sbr_storefront_tile_bed_750x375.jpg", venue_id: 3 },
        { name: "Peach", available: true, capacity: 73, hour_rate: 73.93, day_rate: 42.80, img_url: "https://image.shutterstock.com/z/stock-photo-empty-room-107272922.jpg", venue_id: 5 },
        { name: "Purple", available: true, capacity: 24, hour_rate: 24.28, day_rate: 170.20, img_url: "http://ghk.h-cdn.co/assets/cm/15/11/54ff8227d6954-living-room-gold-orange-xln.jpg", venue_id: 6 },
        { name: "Green", available: false, capacity: 19, hour_rate: 32.53, day_rate: 100.00, img_url: "http://cdn.home-designing.com/wp-content/uploads/2009/04/kids-study-room-skylight.jpg", venue_id: 1 },
        { name: "White", available: true, capacity: 28, hour_rate: 53.86, day_rate: 89.29, img_url: "https://www.gansevoorthotelgroup.com/content/slides/MEATPACKING-Superior-Room31.jpg", venue_id: 4},
        { name: "Rose", available: true, capacity: 53, hour_rate: 24.24, day_rate: 48.38, img_url: "https://s3.amazonaws.com/movoto-foundation/Foundation+Articles/2016/03/living-room-ideas-9-genius-ways-to-create-space/living-room-ideas-9-genius-ways-to-create-space-2.jpg", venue_id: 7 },
        { name: "Teal", available: true, capacity: 24, hour_rate: 18.19, day_rate: 24.19, img_url: "https://static1.squarespace.com/static/537261a3e4b06f7436cc8d37/t/53737004e4b0a52017d8d1bb/1400074245635/Great+Room.jpg", venue_id: 3 },
        { name: "Blue", available: true, capacity: 33, hour_rate: 30.24, day_rate: 73.20, img_url: "http://cdn.home-designing.com/wp-content/uploads/2010/12/beautiful-children-room-ideas.jpg", venue_id: 3 },
        { name: "Neon", available: false, capacity: 24, hour_rate: 24.94, day_rate: 25.73, img_url: "http://amari.azureedge.net/watergate/hotel-photos/deluxe-room-2.jpg", venue_id: 1 },
        { name: "Grey", available: true, capacity: 74, hour_rate: 39.63, day_rate: 130.53, img_url: "http://clv.h-cdn.co/assets/15/15/1428596092-home-sweet-home-living-room-0515.jpg", venue_id: 4 },
        { name: "Colbalt", available: true, capacity: 22, hour_rate: 24.35, day_rate: 140.23, img_url: "https://st.hzcdn.com/fimgs/27f1947b0426b6ea_1649-w500-h400-b0-p0--traditional-dining-room.jpg", venue_id: 6 },
      ]);
    });
};
